docs/src/research/run_win_vm.md

```diff
@@ -32,14 +30,55 @@ You can run Windows 11 in a VM on Ghaf with NVIDIA Jetson Orin AGX. This method
 
     > **WARNING:** Do not use **sudo** or the root user to run windows-launcher.
 
-    Alternatively, you can launch the Windows 11 VM by clicking the corresponding icon in the Weston taskbar.
+2. Windows 11 requires Internet access to finish the setup. To boot the VM without an Internet connection, open cmd with Shift+F10 and type `OOBE\BYPASSNRO`. It will restart configuration and next time there will be a new button “I don’t have internet“. Hit that button to skip connecting to network and continue the installation.
+
+    > **Warning:** Sometimes when Shift+F10 is pressed the command window opens in the background and not visible. In order to bring it to foreground hit Alt+Tab one or more times.
+
+## Running Windows 11 in VM on Generic x86_64 Device
+
+1. On x86_64 device Windows 11 VM can be launched with either an ISO image or QCOW2. When the script launched with an ISO image it creates an empty QCOW2 image in the same directory which is used as a system disk in the VM. After Windows 11 has been installed the script should be launched with the QCOW2 image.
+2. In Weston terminal, go to the directory with the Windows 11 image and run the VM using the following Ghaf script:
+
+    ```
+    cd /mnt
+    windows-launcher ./Win11_22H2_English_x64v2.iso
+    ```
+
+    > **WARNING:** Do not use **sudo** or the root user to run windows-launcher.
+
+3. When the VM starts booting press any key to boot from CD.
+4. In order to bypass Windows 11 system requirement hit Shift+F10 to open a command prompt and type regedit. In `HKEY_LOCAL_MACHINE\SYSTEM\Setup` create a new `LabConfig` key (with right click select New, Key, type LabConfig). In this key create a DWORD (32-bit) parameter with the name `BypassTPMCheck` and value `1`. Also create a similar value with the `BypassSecureBootCheck` name.
```

Comment:
Replace with:

3. In order to bypass Windows 11 system requirements, open cmd with Shift+F10 and type `regedit`. In HKEY_LOCAL_MACHINE\SYSTEM\Setup, right-click _New > Key_ and type _LabConfig_. For this key create two DWORD (32-bit) parameters:
    * Name: `BypassTPMCheck`, value `1`.
    * Name: `BypassSecureBootCheck`, value `1`.